rtio/github-actions-101
# Add deploy workflow

### What and why? 🤔

Add deploy workflow

### Testing Steps ✍️

- All checks passing
- You can add the entry below on `prod.yml`:

```yml
on:
  ...
  pull_request:
    branches:
      - main
```

Adding this will trigger the deployment of the current branch to the GitHub Pages. Be careful!

- Check the changes on: https://rtio.github.io/github-actions-101/.

### You're it! 👑
